internal/exeutils/exeutils_windows_test.go

```diff
@@ -0,0 +1,29 @@
+// +build windows
+
+package exeutils
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_PathForExecutables(t *testing.T) {
```

Comment:
Same as the other test; I think we can implement this without relying on filesystem or env.

Reply:
I can do that.  Two concerns though:
1. In this case, especially when we deal with Windows, I prefer to also test how the file system functions operate.  When abstracting this, I might miss some important behavior...
2. Performance: Gathering all the potential paths, looks at more files and directories than necessary.  Usually not a big problem, but in the executors, performance is a priority of course, even if it is just a few milliseconds...

Reply:
Those are good points. But I'd argue if we really want automated tests to cover those things they should be integration tests.